### PR TITLE
Ansible 6.0.0a2: Dependencies, changelog and porting guide

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

**Comment by dmsimard** [comment]
Marking as draft while we validate and justify version changes for some collections.

File: 6/ansible-6.build [hunk 1]
```diff
@@ -48,30 +48,30 @@ community.routeros: >=2.0.0,<3.0.0
...
 hetzner.hcloud: >=1.6.0,<2.0.0
 hpe.nimble: >=1.1.0,<2.0.0
-ibm.qradar: >=1.0.0,<2.0.0
+ibm.qradar: >=2.0.0,<3.0.0
 infinidat.infinibox: >=1.3.0,<2.0.0
 infoblox.nios_modules: >=1.2.0,<2.0.0
 inspur.sm: >=2.0.0,<3.0.0
-junipernetworks.junos: >=2.10.0,<3.0.0
+junipernetworks.junos: >=3.0.0,<4.0.0
 kubernetes.core: >=2.3.0,<3.0.0
```
Comment (dmsimard) [inline 6/ansible-6.build:75]: Note:
```
WARNING: found collection dependency errors!
WARNING: community.okd version_conflict: kubernetes.core-2.3.1 but needs >=2.1.0,<2.3.0
```

File: 6/galaxy-requirements.yaml [hunk 2]
```diff
@@ -185,13 +179,13 @@ collections:
...
   source: https://galaxy.ansible.com
-  version: 1.15.0
+  version: 1.16.0
 - name: fortinet.fortimanager
   source: https://galaxy.ansible.com
   version: 2.1.5
 - name: fortinet.fortios
   source: https://galaxy.ansible.com
-  version: 2.1.4
+  version: 2.1.5
```
Comment (dmsimard) [inline 6/galaxy-requirements.yaml:188]: ``fortinet.fortimanager`` and ``fortinet.fortios`` may require manual pinning to an earlier version that passes sanity tests without syntax errors, see:
- https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/issues/181
- https://github.com/fortinet-ansible-dev/ansible-galaxy-fortimanager-collection/issues/44

File: 6/ansible-6.build (hunk 1 above)
Comment (felixfontein) [reply 6/ansible-6.build:75]: This must be restricted to < 2.3.0. Can you please fix it?

File: 6/galaxy-requirements.yaml (hunk 2 above)
Comment (felixfontein) [reply 6/galaxy-requirements.yaml:188]: This should be reverted to 2.1.4 as long as their 2.1.5 release is broken.

**Comment by dmsimard** [comment]
The commit to update the GHA jobs will be split to another PR.

In the meantime, I am delaying the release of 6.0.0a2 until we have time to discuss it at wednesday's community meeting.

File: 6/ansible-6.build [hunk 3]
```diff
@@ -82,19 +82,19 @@ netapp.ontap: >=21.18.0,<22.0.0
...
 ngine_io.vultr: >=1.1.0,<2.0.0
 openstack.cloud: >=1.8.0,<2.0.0
 openvswitch.openvswitch: >=2.1.0,<3.0.0
 ovirt.ovirt: >=2.0.0,<3.0.0
-purestorage.flasharray: >=1.12.0,<2.0.0
+purestorage.flasharray: >=1.13.0,<2.0.0
 purestorage.flashblade: >=1.9.0,<2.0.0
 sensu.sensu_go: >=1.13.0,<2.0.0
 servicenow.servicenow: >=1.0.0,<2.0.0
 splunk.es: >=1.0.0,<2.0.0
```
Comment (dmsimard) [inline 6/ansible-6.build:96]:
```
[WARNING]: Collection splunk.es does not support Ansible version 2.13.0rc1
```

File: 6/ansible-6.build [hunk 4]
```diff
@@ -48,30 +48,30 @@ community.routeros: >=2.0.0,<3.0.0
...
-dellemc.openmanage: >=5.2.0,<6.0.0
+dellemc.openmanage: >=5.3.0,<6.0.0
 dellemc.os10: >=1.1.0,<2.0.0
 dellemc.os6: >=1.0.0,<2.0.0
 dellemc.os9: >=1.0.0,<2.0.0
-f5networks.f5_modules: >=1.15.0,<2.0.0
+f5networks.f5_modules: >=1.16.0,<2.0.0
 fortinet.fortimanager: >=2.1.0,<3.0.0
 fortinet.fortios: >=2.1.0,<3.0.0
 frr.frr: >=1.0.0,<2.0.0
```
Comment (dmsimard) [inline 6/ansible-6.build:65]:
```
[WARNING]: Collection frr.frr does not support Ansible version 2.13.0rc1
```

File: 6/ansible-6.build (hunk 1 above)
Comment (dmsimard) [reply 6/ansible-6.build:75]: There could be a new release of community.okd with improved support soon, let's chat about this at the community meeting tomorrow.

**Comment by felixfontein** [comment]
The community.general version should be 5.0.0-a1. I created a fix for the build role here: https://github.com/ansible-community/antsibull/pull/421

File: 6/ansible-6.build (hunk 3 above)
Comment (dmsimard) [reply 6/ansible-6.build:96]: splunk.es just released version 2.0.0: https://github.com/ansible-collections/splunk.es/releases/tag/2.0.0
Will be updated when I rebuild 6.0.0a2.

File: 6/ansible-6.build (hunk 4 above)
Comment (dmsimard) [reply 6/ansible-6.build:65]: There is a version 1.0.4 of frr.frr planned to be out soon with a fixed ansible requirement.

File: 6/ansible-6.build (hunk 1 above)
Comment (dmsimard) [reply 6/ansible-6.build]: The new version of community.okd did not make it in time. We're pinning k8s.core as a result.

File: 6/galaxy-requirements.yaml (hunk 2 above)
Comment (dmsimard) [reply 6/galaxy-requirements.yaml:188]: Done (same as ansible 5.7.1 released earlier today)

File: 6/ansible-6.build (hunk 4 above)
Comment (dmsimard) [reply 6/ansible-6.build:65]: The new version didn't make it out in time for the rebuild of 6.0.0a2. It should be in the next alpha release.

**Review by felixfontein — APPROVED** [review]

**Comment by dmsimard** [comment]
No critical issues when doing sanity tests, it's good to go. Thanks for reviewing @felixfontein !

**Pull request merged**